libs/blocks/card/cardUtils.js

```diff
@@ -48,7 +48,8 @@ export const addWrapper = (el, section, cardType) => {
   const idx = list.findIndex((i) => i.includes(upClass));
   if (idx > -1) {
     upClass = `${idx + 2}-up`;
-    section.classList.remove(list[idx]);
+    const classToRemove = list[idx];
+    new MutationObserver(() => { if (section.classList.contains(classToRemove)) section.classList.remove(classToRemove); }).observe(section, { attributes: true, attributeFilter: ['class'] });
```

Comment:
this looks like hacking the section block. It feels this must be addressed in the section block it-self.
accessing a class name by index and removing it might unexpected issues in the future when the section block code changes.

Reply:
given that, the section is a grid, and you have the following content tree, shouldn't you use `display: contents` on `milo-card-wrapper, consonant-Wrapper-inner, consonant-Wrapper-collection, consonant-CardsGrid` elements
![image](https://github.com/adobecom/milo/assets/330057/49cce188-e537-4ccf-b29e-994494e5028b)